### ISSUE TITLE
Local Development

### ISSUE THREAD

**Issue by jberm4** [ubilabs/geocomplete#275]
Will this work when developing locally?

**Comment by eljoncurry2** [comment]
Yes, works just fine locally.  Either add localhost to your api credentials or leave it empty.

**Comment by ninjz** [comment]
It seems to not be working when developing locally.. I have not been able to find a solution. It looks as if others have been experiencing this issue as well.
